internal/proxy/lb_policy.go

```diff
@@ -166,9 +169,8 @@ func (lb *LBPolicyImpl) ExecuteWithRetry(ctx context.Context, workload ChannelWo
 
 		client, err := lb.clientMgr.GetClient(ctx, targetNode)
 		if err != nil {
-			log.Warn("search/query channel failed, node not available",
-				zap.Int64("nodeID", targetNode),
-				zap.Error(err))
+			msg := fmt.Sprintf("%s channel failed, node not available", workload.execType)
```

Comment:
what happens is the node crashed?

There should be at least two cases we want to clean shard leader cache
1. node is down
2. ErrChannelNotFound
Currently how do we handle a real node down?